app/helpers/collaborators_helper.rb

```diff
@@ -0,0 +1,9 @@
+module CollaboratorsHelper
```

Comment:
Useful for views involving collaborators - distinguishes between collaborators that are associated with groups and those that are not